icons/web3js/web3-original.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128px" height="128px" viewBox="0 0 128 128" version="1.1">
+<g id="surface1">
```

Comment:
Looks like there's some white background left in the icon. Perhaps you could base your svg on the png used on their website?
![image](https://github.com/devicons/devicon/assets/43886029/f555edde-dec2-4bac-924b-ef307283729c)

Reply:
Here are some logo suggestions based on: [web3/web3.js/assets/logo/web3js.svg](https://github.com/web3/web3.js/blob/8c643c2f51ec96b10993e4905c827a16e297f9a5/assets/logo/web3js.svg?plain=1#L4)

`original`
<img width="256" src="https://github.com/devicons/devicon/assets/76992016/49c53310-dc1c-4eef-85b9-fda89d214631">

`plain`
<img width="256" src="https://github.com/devicons/devicon/assets/76992016/872f8186-cc93-4f7e-a710-db0efffe2a22">